packages/grid/src/vaadin-grid-drag-and-drop-mixin.js

```diff
@@ -170,6 +175,8 @@ export const DragAndDropMixin = (superClass) =>
             .filter((row) => !this.dragFilter || this.dragFilter(this.__getRowModel(row)));
         }
 
+        this.__draggedItems = rows.map((row) => row._item);
```

Comment:
The line below that repeats `draggedItems: rows.map((row) => row._item)` could now be changed to `draggedItems: [...this.__draggedItems ]`